lib/twilio_send_sms.rb

```diff
@@ -0,0 +1,13 @@
+require 'time'
+require 'rubygems'
+require 'twilio-ruby'
+
```

Comment:
Seems like you're using the Twilio gem as intended, though it would have been good to inject it into a "wrapper" class (I made a "Phone" class when I did this. In that sense, the actual Twilio object (Twilio::REST::Client.new) is like the antennae of the phone class).